vimiv/config/configfile.py

```diff
@@ -140,3 +145,20 @@ def _read_plugins(pluginsection):
     """
     _logger.debug("Plugins in config: %s", quotedjoin(pluginsection))
     plugins.add_plugins(**pluginsection)
+
+
+def _add_metadata(configsection):
+    """Set available metadata sets from config file.
+
+    Args:
+        configsection: METADATA section in the config file.
+    """
```

Comment:
The use of the regular expression is perfect and very clear :+1: 

I see an issue with the current implementation though: This always removes the defaults, no matter if there is anything in the configfile, as the keysets are always overridden. It also doesn't allow keeping some defaults, and overriding others.

With the dictionary as discussed before, this could be simplified somewhat along:
```
if match:
    number = int(match.group(1))
    api.settings.metadata.keysets[number] = value
    _logger.debug("Keyset %d: '%s' found", number, value)
```